CMakeLists.txt

```diff
@@ -52,6 +52,9 @@ option(FIREBASE_INCLUDE_INSTALLATIONS
 option(FIREBASE_INCLUDE_MESSAGING
        "Include the Firebase Cloud Messaging library."
        ${FIREBASE_INCLUDE_LIBRARY_DEFAULT})
+option(FIREBASE_INCLUDE_PERFORMANCE
+       "Include the Firebase Performance library."
+       ${FIREBASE_INCLUDE_LIBRARY_DEFAULT})
```

Comment:
Could you make the default setting false for this? Since it's not officially supported, users should have to explicitly opt in to include this module (at least for now).